internal/api/lib.go

```diff
@@ -849,6 +849,48 @@ func IBCDestinationCallback(
 	return copyAndDestroyUnmanagedVector(res), convertGasReport(gasReport), nil
 }
 
+func EUPacketReceive(
+	cache Cache,
+	checksum []byte,
+	env []byte,
+	packet []byte,
```

Comment:
The naming seems a bit inconsistent. It's not just the packet, but the full message right?